app/webpacker/components/wca/FormBuilder/input/MarkdownEditor.js

```diff
@@ -100,8 +103,8 @@ function getOptions(disabled) {
     spellChecker: false,
     promptURLs: true,
     previewRender,
-    status: ['upload-image'],
-    uploadImage: true,
+    status: [imageUploadEnabled && UPLOAD_IMAGE_KEY].filter(Boolean),
```

Comment:
I see that this and line 74 are doing the same thing. But I couldn't find a good variable name for both in common, mainly because both are different use-cases - one is toolbar, and other is editor status.